File: trees/tree_data_structure.py
class node(object):
	"""docstring for node"""
	def __init__(self, arg):
		super(node, self).__init__()
		self.val = arg
		self.left = None
		self.right=None

root = node(1)
root.left = node(2)
root.right = node(3)
root.left.left = node(4)
root.left.right = node(5)

File: trees/tree_data_structure.cpp
#include <iostream>
using namespace std;

class Node
{
public:
	Node *left,*right;
	int key;
	Node(int key)
	{
		this->key=key;
		left=NULL;
		right = NULL;
	}
	~Node();
	
};


int	main(int argc, char const *argv[])
{
	Node *root = new Node(1);
	root->left = new Node(2);
	root->right = new Node(3);
	cout<<root->key<<" "<<root->left->key<<" "<<root->right->key<<endl;
	return 0;
}